runtime/resource.go

```diff
@@ -26,15 +27,30 @@ import (
 // 	return false
 // }
 
-func getServerBinary(platform string) string {
+func getServerBinary(platform, cacheDir, version string) (binary string) {
+	pkg, err := FindPackage(cacheDir, version)
+	if err != nil {
+		return
+	}
+
+	var paths map[string]string
 	switch platform {
 	case "windows":
-		return "samp-server.exe"
+		paths = pkg.Win32Paths
 	case "linux", "darwin":
-		return "samp03svr"
+		paths = pkg.LinuxPaths
 	default:
-		return ""
+		return
+	}
+
+	for _, destination := range paths {
+		if strings.Contains(destination, "svr") {
```

Comment:
What's the purpose of this loop and check?

Reply:
This is your code that you posted from this issue:
https://github.com/Southclaws/sampctl/issues/288